examples/paho-mqtt/Makefile

```
@@ -16,7 +16,17 @@ QUIET ?= 1
 
 USEPKG += paho-mqtt
 
+CFLAGS += -DESP_WIFI_SSID=\"define yours\" -DESP_WIFI_PASS=\"secret\"
+
+ifeq ($(BOARD),esp32-wrover-kit)
+USEMODULE += esp_gdb esp_idf_heap esp_log_tagged esp_log_startup esp_log_colored
+USEMODULE += esp_wifi
```

Please indent with 2 spaces.
```suggestion
  USEMODULE += esp_wifi
```